build.gradle

```diff
@@ -101,6 +103,7 @@ dependencies {
     compile 'com.opencsv:opencsv:3.4'
     compile 'com.google.guava:guava:18.0'
     compile 'com.github.samtools:htsjdk:2.5.0'
+    compile 'com.google.cloud:gcloud-java-nio:0.2.5:shaded'
```

Comment:
Add comment here explaining why we are specifically using the `shaded` version of this dependency.